source/whats-new.txt

```diff
@@ -26,6 +27,52 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-4.11:
+
+What's New in 4.11
+------------------
+
+The 4.11 {+driver-short+} release includes **recursive schema support**
+and provides type safety for dot notation queries up to a depth of
+nine. Beyond a depth of nine, your code still compiles but TypeScript no longer
+type checks and values fall back to a type of ``any``. The recursive type
+depth limit is a current limitation in TypeScript.
+
+Feature Demonstration
```

Comment:
Suggestion: describe what you're showing with the subheader text: "<specific feature / support> Example (or Examples)"